packages/@tinacms/cli/src/cmds/compile/index.ts

```diff
@@ -50,9 +52,13 @@ const cleanup = async ({ tinaTempPath }: { tinaTempPath: string }) => {
 export const compile = async (
   _ctx,
   _next,
-  options = { schemaFileType: 'ts' }
+  options: { schemaFileType?: string; verbose?: boolean }
 ) => {
-  logger.info(logText('Compiling...'))
+  if (!options.schemaFileType) options = { ...options, schemaFileType: 'ts' }
+
+  if (options.verbose) {
```

Comment:
@mittonface We could use the logger for this (it does not really matter to me).

It would look something like this.
```ts
if(options.verbose) {
  // set the logger level to verbose
  logger.setLevel('verbose')
}
// instead of logger.into
logger.verbose(logText('Compiling...'))
```

This is more sudo-code (I am not 100% sure of the logger interface)